final_prep/mod_zero_hero.rb

```diff
@@ -44,28 +62,57 @@
 #Test Cases
 announcement = 'Never fear, the Courageous Curly Bracket is here!'
 excuse = 'I think I forgot to lock up my 1992 Toyota Coralla. Be right back.'
-# assess_situation(99, announcement, excuse) > Should print - 'I think I forgot to lock up my 1992 Toyota Coralla. Be right back.'
+#assess_situation(99, announcement, excuse) > Should print - 'I think I forgot to lock up my 1992 Toyota Coralla. Be right back.'
 #assess_situation(21, announcement, excuse) > should print - 'Never fear, the Courageous Curly Bracket is here!'
 #assess_situation(3, announcement, excuse) > should print - "Meh. Hard pass."
-
+assess_situation(99, announcement, excuse)
+assess_situation(21, announcement, excuse)
+assess_situation(3, announcement, excuse)
 # Declare a new variable - scary_monster - assigned to an hash with the following key/values
 #   - name (string)
 #   - smell (string)
 #   - weight (integer)
 #   - citiesDestroyed (array)
 #   - luckyNumbers (array)
 #   - address (hash with following key/values: number , street , state, zip)
-
+scary_monster = {name:"Becky", smell:"Stinky",weight:20,citiesDestroyed:["Denver","San Francisco","New York City"],luckyNumbers:[1,4,6,11],address:{number:433,street:"Broadway",state:"IL",zip:"60657"}}
```

Comment:
This hash would be much easier to read if the key/value pairs were formatted on different lines:

```
scary_monster = {
  name: "Becky",
  smell: "Stinky",
  weight: 20,
  citiesDestroyed: ["Denver","San Francisco","New York City"],
  luckyNumbers: [1,4,6,11],
  address: {
    number: 433,
    street: "Broadway",
    state: "IL",
    zip: "60657"
  }
}
```